### PR TITLE
Revert "Enforce python3 for the dockerfile"

### PR DESCRIPTION
**Pull request by scherniavsky**
Reverts zalando-incubator/docker-locust#87

### CODE REVIEW

**Comment by scherniavsky** [comment]
:+1:

**Comment by thilp** [comment]
:+1:

**Comment by scherniavsky** [comment]
:+1:

**Comment by thilp** [comment]
:+1:

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
-FROM registry.opensource.zalan.do/stups/ubuntu:18.04-15
+FROM registry.opensource.zalan.do/stups/ubuntu
```
Comment (butomo1989) [inline Dockerfile]: As far as I remember, there is an open source python3 base image that Zalando provides instead of using ubuntu image. :b

**Pull request merged**